frontend/src/components/organisms/SearchForm/SearchForm.js

```diff
@@ -110,7 +192,101 @@ class SearchForm extends Component {
 
   render() {
     const { Option } = Select;
+    const { Text } = Typography;
     const countryList = [];
+    const roaList = [];
+    const recruitmentList = [];
+    const phaseList = [];
+    const resultsList = [];
+    const typeList = [];
+    const sexList = [];
+    const ageGroupList = [];
+    const ethnicitiesList = [];
+    const distanceList = [];
+    const statesList = [];
+
+    const recruitmentCheckboxes = [];
+    const phasesCheckboxes = [];
+
+    Array.from(recruitment.entries()).forEach(([index, value]) => {
+      recruitmentCheckboxes.push(
+        <Col key={index} span={12}>
+          <Checkbox key={index} value={value} style={{ lineHeight: '32px' }}>
+            {value}
+          </Checkbox>
+        </Col>
+      );
+    });
+
+    Array.from(phases.entries()).forEach(([index, value]) => {
+      phasesCheckboxes.push(
+        <Col key={index} span={12}>
+          <Checkbox key={index} value={value} style={{ lineHeight: '32px' }}>
+            {value}
+          </Checkbox>
+        </Col>
+      );
+    });
+
+    Array.from(roa.entries()).forEach(([index, value]) => {
+      roaList.push(
+        <Option key={index} value={value}>{value}</Option>
+      );
+    });
+
+    Array.from(recruitment.entries()).forEach(([index, value]) => {
+      recruitmentList.push(
+        <Option key={index} value={value}>{value}</Option>
+      );
+    });
+
+    Array.from(phases.entries()).forEach(([index, value]) => {
+      phaseList.push(
+        <Option key={index} value={value}>{value}</Option>
+      );
+    });
+
+    Array.from(results.entries()).forEach(([index, value]) => {
+      resultsList.push(
+        <Option key={index} value={value}>{value}</Option>
+      );
+    });
+
+    Array.from(types.entries()).forEach(([index, value]) => {
+      typeList.push(
+        <Option key={index} value={value}>{value}</Option>
+      );
+    });
+
+    Array.from(sex.entries()).forEach(([index, value]) => {
```

Comment:
Could this be simplified with a function that takes in entries and then generates this option element rather than repeating the code?

Reply:
Done